EssentialApp/EssentialApp/FeedViewAdapter.swift

```diff
@@ -8,17 +8,21 @@ import EssentialFeediOS
 
 final class FeedViewAdapter: FeedView {
 	private weak var controller: FeedViewController?
+	private let onImageSelection: (FeedImage)->()
```

Comment:
Follow the codebase convention for the closure type. Please, fix it *everywhere*

```suggestion
	private let onImageSelection: (FeedImage) -> Void
```